controllers/discovery/initDiscoveryUserData.js

```diff
@@ -9,11 +9,17 @@ const {QueryTypes} = require('sequelize');
  * @returns
  */
 const InitDiscoveryUserData = async (req, res) => {
-  const {limit = 50, page = 0} = req.body;
+  const {limit = 50, page = 0} = req.query;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `limit` and `page` parameters are now being read from the query string instead of the request body. This is a breaking change if there are any existing clients that send these parameters in the body. If this change is intentional, make sure to update all client-side code and documentation accordingly.
```diff
-  const {limit = 50, page = 0} = req.body;
+  const {limit = 50, page = 0} = req.query;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->